testing/supporting_docs/appium_information.md

```diff
@@ -0,0 +1,69 @@
+## Appium 
+Appium is a cross platform UI testing tool for mobile applications running on 
+either android or iOS devices.  It can be used to test native applications, 
+hybrid applications like Cordova and Xamarin, and web app testing through mobile 
+device browsers.  One of major points of the Appium philosophy is to avoid time 
+consuming processes like compiling to run UI tests.
+
+
+### Appium Locator/Selector Strategies
+
+![alt text](./images/appium_selector_strategies.png "Appium Selector Strategies")
+https://appium.io/docs/en/commands/element/find-elements/#selector-strategies
+
+The picture above shows the available selector strategies to appium in order to 
+locate elements in a mobile application. Note that some of the descriptions also 
+shows the required attributes in for both Android and iOS applications.  It can 
```

Comment:
@twnolan consistency should be persisted with the word 'android' across the whole document, e.g. starting with either lower or upper case.